Sources/TuistGenerator/Generator/SchemesGenerator.swift

```diff
@@ -497,19 +542,22 @@ final class SchemesGenerator: SchemesGenerating {
     /// - Returns: Path to the schemes directory.
     /// - Throws: A FatalError if the creation of the directory fails.
     private func createSchemesDirectory(path: AbsolutePath, shared: Bool = true) throws -> AbsolutePath {
-        let schemePath: AbsolutePath
-        if shared {
-            schemePath = path.appending(RelativePath("xcshareddata/xcschemes"))
-        } else {
-            let username = NSUserName()
-            schemePath = path.appending(RelativePath("xcuserdata/\(username).xcuserdatad/xcschemes"))
-        }
+        let schemePath = schemeDirectory(path: path, shared: shared)
         if !FileHandler.shared.exists(schemePath) {
             try FileHandler.shared.createFolder(schemePath)
         }
         return schemePath
     }
     
+    private func schemeDirectory(path: AbsolutePath, shared: Bool = true) -> AbsolutePath {
+        if shared {
+            return path.appending(RelativePath("xcshareddata/xcschemes"))
+        } else {
+            let username = NSUserName()
```

Comment:
If someone includes its own user data by mistake in the git repository, the wiping logic here won't delete that data. What about deleting any user-related schemes?

Reply:
> If someone includes its own user data by mistake in the git repository, the wiping logic here won't delete that data. What about deleting any user-related schemes?

I might be missing something but I thought https://github.com/tuist/tuist/pull/752/files#diff-d8d6bbd5dbf90ec00b7ea5bd79748c6dR108 was taking caring of this.